src/Gui/Command.cpp

```diff
@@ -963,6 +963,13 @@ Action * Command::createAction(void)
     Action *pcAction;
 
     pcAction = new Action(this,getMainWindow());
+
+#ifdef FC_DEBUG
+    auto cmd = Application::Instance->commandManager().checkAcceleratorForConflicts(sAccel, this);
+    if (cmd)
+        Base::Console().Warning("Accelerator conflict between %s (%s) and %s (%s)\n", sName, sAccel, cmd->sName, cmd->sAccel);
+#endif
+

```

Comment:
As used multiple times, couldn't that be defined as an inline function ?

Reply:
Yeah, probably. I'll do that now.